cmd/agent/app/run.go

```diff
@@ -333,6 +334,16 @@ func StartAgent() error {
 	// create and setup the Autoconfig instance
 	common.LoadComponents(common.MainCtx, config.Datadog.GetString("confd_path"))
 
+	// start the cloudfoundry container tagger
+	if config.IsFeaturePresent(config.CloudFoundry) && !config.Datadog.GetBool("cloud_foundry_buildpack") {
+		containerTagger, err := containertagger.NewContainerTagger()
+		if err != nil {
+			log.Errorf("failed to start Cloud Foundry container tagger: %v", err)
+		} else {
+			containerTagger.Start(common.MainCtx)
+		}
+	}
```

Comment:
Does it make sense to have this logic in `common.LoadComponents`. We init the tagger there already https://github.com/DataDog/datadog-agent/blob/63a5cd88fbf2cc4cc21819a6624274e8c614b261/cmd/agent/common/loader.go#L26-L35

Reply:
I thought about it, but I hesitated because the container tagger is not a `common` component.